src/main/java/com/laborhelper/enterprise/LaborHelperController.java

```diff
@@ -2,7 +2,7 @@
 
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
+
```

Comment:
Removed this import as it wasn't in use.
https://martinfowler.com/bliki/TechnicalDebt.html

Reply:
Use Control-Shift-O to have the IDE organize imports for you.